### PR TITLE
Bug-fix metadata and improves

### PR DESCRIPTION
**Pull request by miiu96**
- Do not transform the characters from the `metadata`  of a NFT in lower-case
- Added `currentOwner` in the AcountsESDT structure in order to index also this information
- Stop indexing  `metaESDT`  created tokens in the `tokens` index
- Refactored some integration-tests to use files with expected json results instead of constants
- Check if an address with a role is already indexed

### CODE REVIEW

**Review by bogdan-rosianu — DISMISSED** [review]

**Review by dragos-rebegea — APPROVED** [review]

**Review by dragos-rebegea — DISMISSED** [review]

File: converters/code.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+		for (i = 0; i < ctx._source.roles.get(params.role).length; i++) {
+			if (ctx._source.roles.get(params.role).get(i) == params.address) {
+				return;
+			}
+		}
+		ctx._source.roles.get(params.role).add(params.address);
+	}
+` */
+// OUTPUT: `if (!ctx._source.containsKey('roles')) {ctx._source.roles = new HashMap();}if (!ctx._source.roles.containsKey(params.role)) {ctx._source.roles.put(params.role, [params.address]);} else {int i;for (i = 0; i < ctx._source.roles.get(params.role).length; i++) {if (ctx._source.roles.get(params.role).get(i) == params.address) {return;}}ctx._source.roles.get(params.role).add(params.address);}`
+func FormatJavaCodeStr(code string) string {
```
Comment (bogdan-rosianu) [inline converters/code.go:24]: rename to `FormatPainlessSource`

File: converters/code.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package converters
+
+import "strings"
+
+// FormatJavaCodeStr will remove from the provided java code new lines and white space's
```
Comment (bogdan-rosianu) [inline converters/code.go:5]: `// FormatPainlessSource will clean all new lines and white spaces from the input`

File: process/logsevents/serialize.go [hunk 3]
```diff
@@ -215,19 +215,42 @@ func serializeRoleData(buffSlice *data.BufferSlice, rd *data.RoleData, role stri
 	meta := []byte(fmt.Sprintf(`{ "update" : {"_index": "%s", "_id" : "%s" } }%s`, index, rd.Token, "\n"))
 	var serializedDataStr string
 	if rd.Set {
+		codeToExecute := `
```
Comment (bogdan-rosianu) [inline process/logsevents/serialize.go:218]: could leave a TODO to change this way everywhere in the project

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: converters/code.go (hunk 1 above)
Comment (miiu96) [reply converters/code.go:24]: done

File: converters/code.go (hunk 2 above)
Comment (miiu96) [reply converters/code.go:5]: done.

File: process/logsevents/serialize.go (hunk 3 above)
Comment (miiu96) [reply process/logsevents/serialize.go:218]: done

**Review by bogdan-rosianu — APPROVED** [review]

**Review by dragos-rebegea — APPROVED** [review]

**Pull request merged**